pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -460,11 +463,11 @@ func gcListWatcher(client *dynamic.Client, resource unversioned.GroupVersionReso
 	}
 }
 
-func monitorFor(p *Propagator, clientPool dynamic.ClientPool, resource unversioned.GroupVersionResource) (monitor, error) {
+func (gc *GarbageCollector) monitorFor(resource unversioned.GroupVersionResource) (monitor, error) {
```

Comment:
In the previous version, the input parameter `p` is a struct defined in GC, so I think it makes sense to convert this function to method of GC.